### ISSUE TITLE
Do not use peerDependencies in package.json

### ISSUE THREAD

**Issue by zerkalica** [ColCh/isparta-instrumenter-loader#3]
npm -v: 2.7.4
node -v: 0.12.2

see https://github.com/npm/npm/issues/6565

```
$ npm i --save-dev isparta-instrumenter-loader
npm WARN peerDependencies The peer dependency isparta@^2.0.0 included from isparta-instrumenter-loader will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
```

**Comment by ColCh** [comment]
Thanks for the issue. 
As I understood that conversation correctly, I should leave **webpack** in _peerDeps_ (because this project is a _loader_) and put `isparta` inside of _dependencies_ section.

**Comment by ColCh** [comment]
Closed via 18dba39b382841bba257f3fd8372e2bdcbb652e8
